inheritance/src/main/java/inheritance/Review.java

```diff
@@ -0,0 +1,26 @@
+package inheritance;
+
+// a body, an author, and a number of stars.
+//
+
+public class Review {
+    // point to the appropriate restaurant
+    // constructor and a toString
+    String author;
+    String body;
+    int stars;
```

Comment:
If you want your reviews to point at the restaurant they are about, you can create a variable here: Restaurant restaurant;  Then in the Review constructor, you could pass in an instance of the restaurant. That's not the only way, but it would be good to have some way to track what Restaurant each Review is about.